### PR TITLE
tt-rss-theme-feedly: Init at 1.4.0

### PR DESCRIPTION
**Pull request by dasJ**
###### Motivation for this change

I use the theme every day because the default one looks crappy.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/servers/tt-rss/theme-feedly/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ... }: stdenv.mkDerivation rec {
```
Comment (pSub) [inline pkgs/servers/tt-rss/theme-feedly/default.nix]: I don't think you need `...` here. AFAIK the use of `...` is discouraged as well.

File: pkgs/servers/tt-rss/theme-feedly/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "levito";
+    repo = "tt-rss-feedly-theme";
+    rev = "v${version}";
+    sha256 = "1n5vci84l0wxsd2k90m2x3j8d7y9kz5fqc6fk6y7r568p1cakg9b";
+  };
+
+  installPhase = ''
```
Comment (pSub) [inline pkgs/servers/tt-rss/theme-feedly/default.nix]: You could add `dontBuild = true` to skip the build phase.

**Review by pSub — CHANGES_REQUESTED** [review]

**Comment by xeji** [comment]
@GrahamcOfBorg build tt-rss-theme-feedly

**Comment by dasJ** [comment]
@pSub Fixed

**Review by pSub — APPROVED** [review]

**Comment by xeji** [comment]
@GrahamcOfBorg build tt-rss-theme-feedly

**Pull request merged**